packages/components/src/docs/Migration/ui-tag.mdx

```diff
@@ -8,10 +8,6 @@ When the new `href` attribute is set the `sd-tag` component renders as a link (`
 
 **Hint:** When the `href` attribute is set i.e. the `sd-tag` is rendered as a link (`<a>`), the `disabled` attribute is ignored, as links may not be disabled. To disable the `sd-tag` the `href` attribute has to be removed as well.
 
-## ✨ New attribute: `removable`
```

Comment:
pls see https://github.com/solid-design-system/solid/pull/353/files#r1310122052

Reply:
New Ticket for that: #357

Reply:
@MarcMatthiae pls update your migration guide before merging.